src/gpuocean/utils/GPUOcean_runDeterministic.py

```diff
@@ -88,6 +88,25 @@ def initlonlat2initgpuocean(source_url, lon, lat,norkyst = True, num_cells_x = 1
     return xinit, yinit, int(x0), int(x1), int(y0), int(y1)
 
 
+def plot_initgpuocean(source_url, initx, inity, x0, x1, y0, y1):
```

Comment:
Rename to plot_initdrifters? 
With the current name, I would expect that it plots the initial GPU Ocean state (eta, hu, hv) as initialized from source_url